src/rf/apps/core/models.py

```diff
@@ -222,16 +232,10 @@ class LayerImage(Model):
         default=0,
         help_text='The order which images are layered (starting from 0)'
     )
-    thumb_small = URLField(
-        null=True,
-        blank=True,
-        help_text='80x80 pixels',
-    )
-    thumb_large = URLField(
-        null=True,
-        blank=True,
-        help_text='300x300 pixels',
-    )
+    thumb_small_key = CharField(max_length=255, blank=True, default='',
```

Comment:
We'll have to do more work than that. In the current plan, the original
files are going to be deleted right? So we really won't have **any** good
way to regenerate them at different sizes later.

On Tue, Oct 20, 2015 at 2:25 PM, Kevin DeLoach <notifications@github.com>
wrote:

> In src/rf/apps/core/models.py
> <https://github.com/azavea/raster-foundry/pull/197#discussion_r42534013>:
>
> > @@ -222,16 +232,10 @@ class LayerImage(Model):
> >          default=0,
> >          help_text='The order which images are layered (starting from 0)'
> >      )
> > -    thumb_small = URLField(
> > -        null=True,
> > -        blank=True,
> > -        help_text='80x80 pixels',
> > -    )
> > -    thumb_large = URLField(
> > -        null=True,
> > -        blank=True,
> > -        help_text='300x300 pixels',
> > -    )
> > +    thumb_small_key = CharField(max_length=255, blank=True, default='',
>
> I think we'll have to add a new column if we ever want to change these
> dimensions (once the application is live). Otherwise, we'll have to distort
> all the previously generated thumbnails to fit the new size. I would be in
> favor of making the dimensions part of the column name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/azavea/raster-foundry/pull/197/files#r42534013>.
>



-- 

Laurence Liss
Software Developer
Azavea  |  340 N 12th St., Ste. 402, Philadelphia, PA 19107
E *lliss@azavea.com <lliss@azavea.com>* | T *215.701.7691*  | F
*215.925.2663 *www.azavea.com  |  azavea.com/blogs  |  Twitter @azavea
<http://twitter.com/azavea>